packages/okta-react/test/e2e/harness/src/SessionTokenLogin.js

```diff
@@ -29,9 +29,8 @@ export default withOktaAuth(class SessionTokenLogin extends Component {
   }
 
   componentWillMount() {
-    const origin = new URL(this.props.authService._config.issuer).origin;
     this.oktaAuth = new OktaAuth({
-      url: origin
+      issuer: this.props.authService._config.issuer
```

Comment:
Might make sense to move config to a separate file so it could be shared. Alternatively, you can use `process.env.ISSUER` Even more alternatively, why do we need to create a separate instance of AuthJS here? Why can't we just use `props.authService` ?

Reply:
Good question - I'm not sure why this was using a separate instance in the first place.  I'll take a peek.

Reply:
Looks like this was a separate instance because it's trying to create a different config - just the issuer, not the other params (clientId, redirectUri, onAuthRequired) because this test is explicitly testing the use of the session token.  

Given that this.props.authService._config is set via process.env.ISSUER, I'm inclined to use that rather than introduce env variables to multiple files, unless you're aware of a problem with that.